ansible/playbook/tasks/update.yml

```diff
@@ -18,6 +18,7 @@
       - percona-clickhouse-client
       - percona-clickhouse-server
       - pmm2-client
+      - dbaas-tools
```

Comment:
it should be in pmm-image not here

Reply:
It should be there _too_ – we want to update it during PMM Server update

Reply:
To reiterate: both dbaas-tools and dbaasc-controller should be in two places – there and in percona-images